tox.ini

```diff
@@ -1,6 +1,6 @@
 # tox.ini
 [tox]
-envlist = py27
+envlist = py27, py34
```

Comment:
Btw a good chance to also add corresponding modifiers in [setup.py](https://github.com/scrapy-plugins/scrapy-pagestorage/blob/master/setup.py#L21). Also, I'd also add test environments for python 3.3 and 3.5, it should work the same, isn't it?